apps/couchdb.go

```diff
@@ -112,7 +112,7 @@ func (p LoggingProcessorCouchdb) Components(tag string, uid string) []fluentbit.
 			"Match":         tag,
 			"Operation":     "nest",
 			"Wildcard":      "http_request_*",
-			"Nest_under":    "logging.googleapis.com/http_request",
+			"Nest_under":    confgenerator.HttpRequestKey,
```

Comment:
This is fine for a short-term fix, but this copy-pasting of config components is ridiculous — longer term, we should have a common function that every receiver can call to set this up. Probably not worth taking the time to refactor it now. @qingling128 for thoughts…

Reply:
+1 to having something added here. Every receiver in the Ops Agent (which most 3rd Party integrations are) are expected to create a `LogEntry` since processors down in the pipeline expect that contract. Having some common util here, that would be more easily maintainable that would define and codify that contract would be my preference. But thats orthogonal and non-blocking to this PR.

Reply:
Took a look at `common_logging_processors.go` and what's there is very specific to access logs. We could have some pretty easy wins if we make some common processor utilities a little more universal.

Reply:
I'd vote for deferring this to a follow-up PR.

Reply:
+1

Reply:
I will leave it out of this PR, but I am interested in looking deeper into this.